src/Instagram/Provider.php

```diff
@@ -70,19 +74,21 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'     => $user['id'], 'nickname' => $user['username'],
-            'name'   => $user['full_name'], 'email' => null,
-            'avatar' => $user['profile_picture'],
+            'id'            => $user['id'],
+            'name'          => $user['username'],
+            'account_type'  => $user['account_type'],
+            'avatar'        => $user['profile_picture'] ?? null,
+            'media_count'   => $user['media_count'] ?? null,
         ]);
     }
 
     public function getAccessToken($code)
     {
         $response = $this->getHttpClient()->post($this->getTokenUrl(), [
-            RequestOptions::FORM_PARAMS => $this->getTokenFields($code),
+            'form_params' => $this->getTokenFields($code),
         ]);
 
-        $this->credentialsResponseBody = json_decode((string) $response->getBody(), true);
```

Comment:
Same here, this is correct.

Reply:
This was an oversite and has been reverted back.